src/modelAnalysis/coupling/constrainRxnListAboveBound.m

```diff
@@ -0,0 +1,54 @@
+function modelConstrained=constrainRxnListAboveBound(model,rxnList,c,d,csense)
+%
+% Constrains a (weighted) sum of absolute fluxes to be above a lower bound.
+%
```

Comment:
Isn't whats added just a sum of fluxes (i.e. Sum v_i < k) and not absolute ie Sum |v_i|  < k?